examples/monaco-graphql-react-vite/src/App.tsx

```diff
@@ -1,69 +1,67 @@
-import React, { useEffect } from 'react';
+import { ReactElement, useEffect, useRef, useState, StrictMode } from 'react';
 import { getIntrospectionQuery, IntrospectionQuery } from 'graphql';
 import { Uri, editor, KeyMod, KeyCode, languages } from 'monaco-editor';
 import { initializeMode } from 'monaco-graphql/src/initializeMode';
-import { createGraphiQLFetcher } from '@graphiql/toolkit';
+import { createGraphiQLFetcher, SyncFetcherResult } from '@graphiql/toolkit';
 import * as JSONC from 'jsonc-parser';
 import { debounce } from './debounce';
 
 const fetcher = createGraphiQLFetcher({
-  url: 'https://api.spacex.land/graphql/',
+  url: 'https://countries.trevorblades.com',
```

Comment:
previous endpoint is no longer works

Reply:
oh ok! let's add a default query for this new schema perhaps?

we can also use our own to make sure it continues working

Reply:
I changed the default query too!
https://github.com/graphql/graphiql/blob/bf99ea8e8d4b50c58e934ce3a1bf3b51e0991cdf/examples/monaco-graphql-react-vite/src/App.tsx#L19-L25